.github/ISSUE_TEMPLATE/map_problem.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Codacy found an issue: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6221360)